src/components/validateForm/validateForm.js

```diff
@@ -0,0 +1,23 @@
+export default function validateAuth(values) {
+  let errors = {}
+  //cost
+  if(!values.cost) {
```

Comment:
btw... `required`, `type="number"` and `min="0"` attributes in input tag doesn't prevent those errors?